multiqc/modules/picard/OxoGMetrics.py

```diff
@@ -40,6 +40,20 @@ def parse_reports(self):
                     sample_names.append(s_name)
                     s_files.append(f)
 
+            elif "ConvertSequencingArtifactToOxoG" in l and "INPUT_BASE" in l:
+                s_name = None
+                keys = None
+                context_col = None
+
+                # Pull sample name from input
+                fn_search = re.search(r"INPUT_BASE(?:=|\s+)(\[?[^\s]+\]?)", l, flags=re.IGNORECASE)
```

Comment:
I don't have any example data in ewels/MultiQC_TestData which has `ConvertSequencingArtifactToOxoG` in it so I can't test this. But I think that your change is basically to allow either `INPUT` or `INPUT_BASE` in the regex, is that right? In which case I'm not sure that we need to duplicate this whole code block. I think that we can just add an optional non-capturing group to the regex to allow either.